scripts/camViewer

```diff
@@ -264,14 +264,18 @@ if [ $# -ge 1 ]; then
     exit 1
 fi
 
-
-if [ "$hutch" == ""  ]; then
-    hutch=$(get_info --gethutch)
+hutch_check() {
+hutch=$(get_info --gethutch)
```

Comment:
In general, it's not good practice to make functional changes in commits listed as simple message or documentation changes.
That being said, I like that this makes the error more understandable to the user.
A couple comments though:
- Why did you unindent line 268?
- This block doesn't seem like the right place to print that message on 271 since it will get printed out to the user no matter whether they are running a start/stop command or not. Line 270 and 271 should be used in different cases. Maybe solve with an if/else or moving the echo.

Reply:
1) I don't think it was meant to be indented. I will have changed in the next commits.
2) Will be updated in the next commits.